requirements.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: redis
   repository: https://charts.bitnami.com/bitnami
-  version: 10.6.13
-digest: sha256:b15c80be33efeef65f46fe365ef7a8d2e772b819a052543b94998ca554721bad
-generated: "2022-05-20T17:26:03.8715019+02:00"
+  version: 17.0.10
```

Comment:
Is there a reason not to update to v17.1.4?

Also we should definitely do some data integrity tests during this run since we will bump redis from 6.2 to 7.0.